apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -0,0 +1,105 @@
+/** @file Settings menu cog icon */
+import React, {Component} from 'react';
+import Radium from 'radium';
+import Portal from 'react-portal';
+import msg from '@cdo/locale';
+import FontAwesome from '../../templates/FontAwesome';
+import color from '../../util/color';
+import SettingsMenu from './SettingsMenu';
+
+const style = {
+  iconContainer: {
+    float: 'right',
+    marginRight: 10,
+    marginLeft: 10,
+    height: '100%',
+    cursor: 'pointer',
+    color: color.lighter_purple,
+    ':hover': {
+      color: color.white,
+    }
+  },
+  assetsIcon: {
+    fontSize: 18,
+    verticalAlign: 'middle',
+  },
+};
+
+class SettingsCog extends Component {
+  constructor(props) {
+    super(props);
+
+    // Autobind
```

Comment:
I think marginally more confusing, mostly because this is being done because the class syntax means we don't get a specific feature that is referred to as auto-binding. I worry that someone who hears that we only get autobinding if we use createClass will be confused when they see something called autobinding here.